TorqueDataConnect/extension.json

```diff
@@ -78,5 +80,28 @@
     "torquedataconnect-apiaccess/*": "LogFormatter",
     "torquedataconnect-datachanges/*": "LogFormatter"
   },
+  "ResourceModules": {
+    "ext.torquedataconnect.css": {
+      "styles": "resources/css/TorqueDataConnectEdit.css",
+      "targets": [
+        "desktop",
+        "mobile"
```

Comment:
Curious, have you tested this on mobile?  Does it look good?

Reply:
The template doesn't seem suited to mobile but given that editing of elements happens inline, the experience of editing on mobile is largely the same as desktop.